content/understand-django/2020-01-22-urls-lead-way.es.md

```diff
@@ -0,0 +1,444 @@
+---
+title: "Las URL Marcan El Camino"
+slug: "url-marcan-camino"
+description: >-
+    How does a Django site know
```

Comment:
The description is missing a translation.

Reply:
¿Cómo sabe un sitio de Django dónde enviar solicitudes? ¡Tienes que decírselo!
    En este artículo que continúa la serie Comprendiendo Django, examinaremos las URL y cómo permitir que tus usuarios lleguen al lugar correcto.